embedded-keycloak/src/main/java/com/playtika/testcontainer/keycloak/KeycloakContainer.java

```diff
@@ -14,7 +14,7 @@
 @Slf4j
 public class KeycloakContainer extends GenericContainer<KeycloakContainer> {
 
-    private static final int DEFAULT_HTTP_PORT_INTERNAL = 8080;
+    public static final int KEYCLOAK_DEFAULT_HTTP_PORT_INTERNAL = 8080;
```

Comment:
renamed to be more unique
_(because I'm making it public and don't want to expose such a generic name)_